### PR TITLE
Add Slack icon and workspace link

### PR DESCRIPTION
**Pull request by McPringle**
This change is [Reviewable](https://reviewable.io/reviews/hackergarten/hackergarten.github.io/323)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,12 +142,13 @@
...
                 <div id="joinSection" class="panel-heading">Join</div>
                 <div class="panel-body">
+                    <a href="https://hackergarten.slack.com/" title="Enter Slack Workspace"><img src="pictures/slack.png" alt="Slack Workspace" id="slack" /></a>
                     <ul>
                         <li><a href="feed.xml">Hackergarten Events RSS Feed</a></li>
                         <li><a href="http://github.com/hackergarten">Public Git Repository</a><br>
                         <li><a href="http://www.twitter.com/Hackergarten">@Hackergarten on Twitter</a></li>
                         <li><a href="https://gitter.im/hackergarten?utm_source=share-link&utm_medium=link&utm_campaign=share-link">Gitter</a></li>
-                        <li><a href="https://forms.gle/93guiZ3rXtW4azdo6">Slack Community</a></li>
+                        <li>Slack: <a href="https://forms.gle/93guiZ3rXtW4azdo6">request invitation</a> and/or <a href="https://hackergarten.slack.com/">enter workspace</a></li>
```
Comment (madmas) [inline index.html:151]: Why do we have both - this line and # 145 ? I think just this is sufficient, isn´t it ?

Comment (McPringle) [reply index.html:151]: Line 145 is only the Slack icon (with a workspace link). If you don't want the workspace link in the list, too, I can remove it and just have the "Request Slack invitation" link. Unfortunately, we need both, one link to the invitation form and one to the workspace.

**Comment by McPringle** [comment]
Hi @madmas, what's about this PR? Still not satisfied with my answer to your review comment?

**Comment by madmas** [comment]
@McPringle  thanks a lot for the heads up :-)

**Pull request merged**